README.md

```diff
@@ -35,6 +35,7 @@ options:
   -t TARGET_PATH, --target TARGET_PATH                                                                                                                  choose single target image or video
   -o OUTPUT_PATH, --output OUTPUT_PATH                                                                                                                  specify the output file or directory
   -v, --version                                                                                                                                         show program's version number and exit
+  -i INI_PATH, --ini INI_PATH                                                                                                                           specify ini file for launch configuration
```

Comment:
move it to the very top position

Reply:
`-c CONFIG_PATH, --config CONFIG_PATH` 
or 
`-i CONFIG_PATH, --ini CONFIG_PATH`  ?